ukat/segment/tests/__init__.py

```diff
@@ -0,0 +1 @@
+from . import test_whole_kidney
```

Comment:
Please also add "segment" to the import in `ukat/__init__.py`

Reply:
Now I realised that "segment" is the name of the folder but also the name of the renalsegmentor package. Would renaming the folder "segmentation" be a better option to avoid confusion?

Reply:
Yeah, makes sense.